### ISSUE TITLE
[BUG] GPU Graph is wierdly filled and not representing the real load

### ISSUE THREAD

**Issue by sibwaze** [orblazer/plasma-applet-resources-monitor#84]
### Current Behavior

As far as I understand and remember it was on previous versions on KDE Plasma 5, GPU load graph should illustrate the ratio between the current load and 100% (fully filled), but it is always showing almost full load for some reason

### Expected Behavior

Fully filled on full load
Not filled on no load
Half filled on half load, etc.

### Steps To Reproduce

Just install the applet/widget and add your GPU

### Screenshots

![изображение](https://github.com/orblazer/plasma-applet-resources-monitor/assets/98026017/881e0d16-6e3e-4c59-813b-ee6f3731857c)

### Logs

```Text
мая 11 20:31:20 sibwase plasmashell[2027]: file:///usr/share/plasma/shells/org.kde.plasma.desktop
/contents/configuration/AppletConfiguration.qml:142:5: QML Connections: Detected function "onSettingValueChanged" in Connections elem
ent. This is probably intended to be a signal handler but no signal of the target matches the name.
мая 11 20:31:33 sibwase plasmashell[2027]: file:///home/sibwase/.local/share/plasma/plasmoids/org
.kde.olib.thermalmonitor/contents/ui/config/ConfigSensors.qml:14:1: QML ConfigSensors: Created graphical object was not placed in the
 graphics scene.
мая 11 20:31:33 sibwase plasmashell[2027]: file:///usr/share/plasma/shells/org.kde.plasma.desktop
/contents/configuration/AppletConfiguration.qml:142:5: QML Connections: Detected function "onSettingValueChanged" in Connections elem
ent. This is probably intended to be a signal handler but no signal of the target matches the name.
мая 11 20:31:35 sibwase plasmashell[2027]: file:///home/sibwase/.local/share/plasma/plasmoids/org
.kde.netspeedWidget/contents/ui/config/ConfigGeneral.qml:22:1: QML ConfigGeneral: Created graphical object was not placed in the grap
hics scene.
мая 11 20:31:35 sibwase plasmashell[2027]: file:///home/sibwase/.local/share/plasma/plasmoids/org
.kde.netspeedWidget/contents/ui/config/ConfigGeneral.qml: Setting initial properties failed: ConfigGeneral does not have a property c
alled cfg_byteColorDefault
мая 11 20:31:35 sibwase plasmashell[2027]: file:///home/sibwase/.local/share/plasma/plasmoids/org
.kde.netspeedWidget/contents/ui/config/ConfigGeneral.qml: Setting initial properties failed: ConfigGeneral does not have a property c
alled cfg_customColorsDefault
мая 11 20:31:35 sibwase plasmashell[2027]: file:///home/sibwase/.local/share/plasma/plasmoids/org
.kde.netspeedWidget/contents/ui/config/ConfigGeneral.qml: Setting initial properties failed: ConfigGeneral does not have a property c
alled cfg_fontSizeDefault
мая 11 20:31:35 sibwase plasmashell[2027]: file:///home/sibwase/.local/share/plasma/plasmoids/org
.kde.netspeedWidget/contents/ui/config/ConfigGeneral.qml: Setting initial properties failed: ConfigGeneral does not have a property c
alled cfg_gigabyteColorDefault
мая 11 20:31:35 sibwase plasmashell[2027]: file:///home/sibwase/.local/share/plasma/plasmoids/org
.kde.netspeedWidget/contents/ui/config/ConfigGeneral.qml: Setting initial properties failed: ConfigGeneral does not have a property c
alled cfg_interfacesWhitelist
мая 11 20:31:35 sibwase plasmashell[2027]: file:///home/sibwase/.local/share/plasma/plasmoids/org
.kde.netspeedWidget/contents/ui/config/ConfigGeneral.qml: Setting initial properties failed: ConfigGeneral does not have a property c
alled cfg_interfacesWhitelistDefault
```

### Versions

```Markdown
- Widget: 3.0.0-rc.1
- KDE Plasma: 6.0.4
- KDE Framework: 6.1
```

**Comment by orblazer** [comment]
Hello,
That seem to be normal for me, if you have game or some other program it load many things in VRAM.
And if is not i think is some driver issue due i use `gpu/gpuX/totalVram` and `gpu/gpuX/usedVram` and it seam both return values, if the fill was bugged, you will see 100% of the height to be filled and doesn't see the solid bar on top.

**Comment by sibwaze** [comment]
I have 4GB of total VRAM, while 2GB is availiable right now, but for some reason widget shows like i am almost full
You can see on my screenshot that System Monitor shows correctly, whilst widget is not
![изображение](https://github.com/orblazer/plasma-applet-resources-monitor/assets/98026017/2c3bdb2a-c50d-41a5-8da7-d596b2986322)

**Comment by orblazer** [comment]
>![изображение](https://private-user-images.githubusercontent.com/98026017/329859513-2c3bdb2a-c50d-41a5-8da7-d596b2986322.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1NjgzNjcsIm5iZiI6MTcxNTU2ODA2NywicGF0aCI6Ii85ODAyNjAxNy8zMjk4NTk1MTMtMmMzYmRiMmEtYzUwZC00MWE1LThkYTctZDU5NmIyOTg2MzIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEzVDAyNDEwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE3NTEyYzI1MDA5MDhhZDI3ZjZmZmU2ZDEwMmZlNjE5OTYwMDlkZjlhMzk4YzUyNTc4NDRjNzk3OGY5MmJmYjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.aG9ZUPdIW3ET0gQPAOMxSS0U5dJM6As1CHVaGnnFtkk)

Could you translate me the labels ? and it is `system monitor` app ?

**Comment by sibwaze** [comment]
> Could you translate me the labels ? and it is `system monitor` app ?

Yes, the first screenshot is the 'system monitor' app and the second is the amdgpu_top
which exact labels you want me to translate to english?
i've shown on the last screenshot that amdgpu_top shows correctly availiable amount of VRAM (total of 4GB) while widget shows that i don't have availiable VRAM even when i have 2GBytes

**Comment by orblazer** [comment]
Hum the first chart and maybe the second, to see what represent the VRam.

And i think the sensor "total vram" show only the "available" one on AMD, could you check in `system monitor` app, by following this step :
1. create an new page
2. select the sensor `GPU X Total Video Memory`
3. see the result

Thanks.

**Comment by sibwaze** [comment]
> hum that weird, could you check if `GPU X Usage` is right value too ? if yes, i think is some issue with init function in my code and i will check if in few days due i finally have plasma 6 on my PC.

yeah, sure, here it is 
![изображение](https://github.com/orblazer/plasma-applet-resources-monitor/assets/98026017/fafafef9-9e8d-4b52-859a-27e44de623a9)
and the gpu used vram
![изображение](https://github.com/orblazer/plasma-applet-resources-monitor/assets/98026017/c842ca75-4e01-427f-96da-d9627d3f8fee)

**Comment by sibwaze** [comment]
Yeah now it works how it should, thanks!
